readme.md

```diff
@@ -261,7 +261,7 @@ const Component = () => {
 Reducing nested structures is tiresome. Have you tried [immer](https://github.com/mweststrate/immer)?
 
 ```jsx
-import produce from 'immer'
+import { produce } from 'immer'
```

Comment:
Okay, it's dropped since v10.
https://github.com/immerjs/immer/releases/tag/v10.0.0